tensorly/decomposition/robust_decomposition.py

```diff
@@ -90,7 +90,10 @@ def robust_pca(X, mask=None, tol=10e-7, reg_E=1, reg_J=1,
     for iteration in range(n_iter_max):
 
         for i in range(T.ndim(X)):
-            J[i] = fold(svd_thresholding(unfold(D, i) + unfold(L[i], i)/mu, reg_J/mu), i, X.shape)
+            svd_thresh = svd_thresholding(unfold(D, i) + unfold(L[i], i)/mu, reg_J/mu)
+            if not T.is_tensor(svd_thresh):
+                svd_thresh = T.tensor(svd_thresh)
```

Comment:
When/why would this happen?
It seems to me that svd_thresholding always returns a tensor.

Reply:
@asmeurer I think that's the only change to the actual Robust PCA here, the rest is already added in other PRs

Reply:
CC @scopatz